Explorer/Assets/DCL/SDKComponents/AudioSources/Systems/UpdateAudioSourceSystem.cs

```diff
@@ -122,5 +133,30 @@ private void HandleSDKChanges(PBAudioSource sdkComponent, ref AudioSourceCompone
 
             sdkComponent.IsDirty = false;
         }
+
+        /// <summary>
+        /// Mutes an AudioSource depending on whether its world position in contained in the current scene.
+        /// If the AudioSource is in the bounding box of the scene, it is un-muted; otherwise it is muted.
+        /// </summary>
+        /// <param name="entity">The entity that contains the AudioSource and its Transform.</param>
+        /// <param name="component">The AudioSource component.</param>
+        [Query]
+        private void UpdateAudioSourceMute(in Entity entity, ref AudioSourceComponent component)
+        {
+            Vector3 audioWorldPosition = World.Get<TransformComponent>(entity).Transform.position;
```

Comment:
`TransformComponent` must be an argument - this is how queries work - with `World.Get<TransformComponent>(entity)` you can't guarantee its presence

Reply:
Oops! you are right!